### PR TITLE
Updated Devexpress Library 17.1

### PR DESCRIPTION
**Pull request by ghost1372**
افزودن پشتیبانی از کامپوننت های Devexpress
نسخه 17.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
-[![Build status](https://ci.appveyor.com/api/projects/status/b6qjwi71527ifh0g/branch/master?svg=true)](https://ci.appveyor.com/project/HEskandari/farsilibrary/branch/master)
+[![Build status](https://ci.appveyor.com/api/projects/status/b6qjwi71527ifh0g/branch/master?svg=true)](https://ci.appveyor.com/project/HEskandari/farsilibrary/branch/master) | [![GitHub version](https://badge.fury.io/gh/ghost1372%2FFarsiLibrary.svg)](https://badge.fury.io/gh/ghost1372%2FFarsiLibrary)
```
Comment (HEskandari) [inline README.md]: Why is this needed? It is pointing to the wrong branch.

File: FarsiLibrary.Win/FarsiLibrary.Win.csproj [hunk 2]
```diff
@@ -12,7 +12,7 @@
     <AssemblyName>FarsiLibrary.Win</AssemblyName>
     <RootNamespace>FarsiLibrary.Win</RootNamespace>
     <OutputPath>..\binaries\</OutputPath>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```
Comment (HEskandari) [inline FarsiLibrary.Win/FarsiLibrary.Win.csproj]: Why downgrade to v4.5?

File: FarsiLibrary.Win.DevExpress.17.1/XtraFADateEdit.TouchUI.cs [hunk 3]
```diff
@@ -0,0 +1,324 @@
+﻿using System;
```
Comment (HEskandari) [inline FarsiLibrary.Win.DevExpress.17.1/XtraFADateEdit.TouchUI.cs]: This whole fine is not needed. Add a link to existing file in 15.1.

File: FarsiLibrary.Win.DevExpress.17.1/Properties/licenses.licx [hunk 4]
```diff
@@ -0,0 +1,2 @@
+DevExpress.XtraEditors.Repository.RepositoryItemDateEdit, DevExpress.XtraEditors.v17.1, Version=17.1.4.0, Culture=neutral, PublicKeyToken=b88d1754d700e49a
```
Comment (HEskandari) [inline FarsiLibrary.Win.DevExpress.17.1/Properties/licenses.licx]: No need to include the license file.

Comment (ghost1372) [reply FarsiLibrary.Win.DevExpress.17.1/Properties/licenses.licx]: ok

File: FarsiLibrary.Win.DevExpress.17.1/XtraFADateEdit.TouchUI.cs (hunk 3 above)
Comment (ghost1372) [reply FarsiLibrary.Win.DevExpress.17.1/XtraFADateEdit.TouchUI.cs]: ok

File: FarsiLibrary.Win/FarsiLibrary.Win.csproj (hunk 2 above)
Comment (ghost1372) [reply FarsiLibrary.Win/FarsiLibrary.Win.csproj]: because of my project so i changed version

File: README.md (hunk 1 above)
Comment (ghost1372) [reply README.md]: ok
